pages/content/ch2/ebnf.mdx

```diff
@@ -0,0 +1,32 @@
+# EBNF
+
+This tries to define formulas and statements with EBNF. Note that all literals are in double quotes.
+
+_Symbol_ $\Leftarrow$ A-Z
+
+_LogicalOperator_
+    $\Leftarrow$ "$\land$"
+        | "$\lor$"
+        | "$\to$"
+        | "$\lnot$"
+
+_FormulaToStatementOperator_
+    $\Leftarrow$ "$\equiv$"
+        | "$\models$"
+
+_StatementOperator_
+    $\Leftarrow$ "$\implies$"
+        | "and"
+        | "or"
+_Formula_
+    $\Leftarrow$ _Symbol_
+        | _Symbol_ _LogicalOperator_ _Symbol_
+        | "("_Formula_ _LogicalOperator_ _Formula_")"
```

Comment:
```suggestion
        | "(" _Formula_ ")"
```

Reply:
huh

Reply:
kürzer und Klammern können gestacked werden.

Reply:
ich glaube man könnte ein Bild erstellen... also einfach in latex tabelle erstellen und dann einfügen maybe

Reply:
oder vllt alles in ein aligned block